internal/fluentbit/config/builder/config_builder.go

```diff
@@ -35,7 +35,7 @@ func BuildFluentBitConfig(pipeline *telemetryv1alpha1.LogPipeline, defaults Pipe
 	sb.WriteString(createKubernetesMetadataFilter(pipeline))
 	sb.WriteString(createLuaDedotFilter(pipeline))
 	sb.WriteString(createOutputSection(pipeline, defaults))
-
+	sb.WriteString(createPrometheusMetricOutput("internal_metrics"))
```

Comment:
This configuration is pipeline specific and would repeat the config snippet per LogPipelien. I think `fluentbit.MakeConfigMap` would be the better place for this.

Reply:
moved